vmm/src/config.rs

```diff
@@ -187,6 +192,41 @@ impl<'a> DiskConfig<'a> {
     }
 }
 
+#[derive(PartialEq)]
+pub enum ConsoleOutputMode {
+    Off,
+    Tty,
+    File,
+}
+
+pub struct ConsoleConfig<'a> {
```

Comment:
This is the same as `SerialConfig` replace all uses of the former with this.

Reply:
Done.